distribution/docs/src/main/resources/content/_tables/catalog.ui.config.adoc

```diff
@@ -114,51 +137,54 @@ TMS example (3d map support only): `{ "name": "Example TMS", "show": true, "type
 
 |Use External Authentication
 |externalAuthentication
-|Use an external authentication point, such as IdP
 |Boolean
+|Use an external authentication point, such as IdP
```

Comment:
✏️ missing period.

Reply:
I also fixed this in the `Standard Search UI` docs. I went ahead and updated the `Standard Search UI` docs/metatype to match `Catalog UI Search`.